recipes/wxdata/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set python_min = '3.10' %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default.

```suggestion
```